gum/README.md

```diff
@@ -44,12 +44,29 @@ To use Disqus, add the Disqus site name via the following variable.
 DISQUS_SITENAME = ''
 ```
 
+To add extra CSS, eg to override styles without forking the Gum theme,
```

Comment:
`eg` can be confusing for folks, particularly non-native English speakers, so may suggest the following phrasing instead?

> To add extra CSS, for example to override styles without forking the Gum theme,